_topic_maps/_topic_map.yml

```diff
@@ -3366,14 +3366,8 @@ Topics:
 - Name: Backup and restore
   Dir: backup_restore
   Topics:
-  - Name: Backup and restore overview
+  - Name: Backing up and restoring VMs
```

Comment:
s/VMs/virtual machines

I think in our docs we have always spelled out virtual machines in the topic title